server/server.go

```diff
@@ -473,7 +472,7 @@ func (s *Server) Run(ctx context.Context) error {
 	}
 
 	httpServer := &http.Server{
-		Addr: fmt.Sprintf(":%d", s.config.HTTPServer.Addr),
+		Addr: fmt.Sprintf(":%d", s.config.HTTPServer.Port),
```

Comment:
Interesting. I guess I never noticed this before. Is `Port` sufficient though? Won't there be some case where a user will want to use the the entire `Addr`?